### PR TITLE
Adding cards for VectorDiJet1Jet signals

### PR DESCRIPTION
**Pull request by cmantill**
Hi, 
I added the cards for producing VectorDiJet1Jet signals for EXO's boosted dijet analysis that uses DMsimp_s_spin1 model. Please let me know if something needs to be modified. I'm currently running a gridpack for M = 125 GeV to test it works.

Thanks,
Cristina

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Vector_Dijet_LO/Vector_Dijet_LO_Mphi/Vector_Dijet_LO_Mphi_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+  0.4	= dparameter 
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+  5	= maxjetflavor ! Maximum jet pdg code
+#*********************************************************************
+# Jet measure cuts                                                   *
+#*********************************************************************
+  0.0	= xqcut ! minimum kt jet measure between partons
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Vector_Dijet_LO/Vector_Dijet_LO_Mphi/Vector_Dijet_LO_Mphi_run_card.dat]: xqcut = 0 is very inclusive and can lead to mg5_amc failures. Which pythia qcut are you using? can you increase a bit the xqcut?

Comment (cmantill) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Vector_Dijet_LO/Vector_Dijet_LO_Mphi/Vector_Dijet_LO_Mphi_run_card.dat]: Changed to xqcut(MG) = 15.0 and I think for 2015 we were using xqcut(Pythia) = 16.0, but I'm not sure about the latter one.

Comment (efeyazgan) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Vector_Dijet_LO/Vector_Dijet_LO_Mphi/Vector_Dijet_LO_Mphi_run_card.dat]: If you want to make sure that the value you use OK, you can follow https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideSubgroupMC#Measure_proper_value_of_qCut

Comment (cmantill) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Vector_Dijet_LO/Vector_Dijet_LO_Mphi/Vector_Dijet_LO_Mphi_run_card.dat]: Thanks @efeyazgan, already changed this to 15 (see updated version). We made these studies before and we want to keep it that way, with xqcut=15.

**Pull request merged**